src/test/java/com/wikia/webdriver/elements/mercury/components/Sidebar.java

```diff
@@ -0,0 +1,147 @@
+package com.wikia.webdriver.elements.mercury.components;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.elements.oasis.pages.TemplateEditPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import java.util.List;
+
+public class Sidebar extends WikiBasePageObject {
+
+  @FindBy(css = ".infobox-builder-sidebar-header .infobox-builder-sidebar-header-icon-delete")
+  private WebElement deleteButton;
+
+  @FindBy(css = ".infobox-builder-sidebar-padding .check-box-input")
+  private WebElement sidebarCheckbox;
+
+  @FindBy(css = ".infobox-builder-sidebar-padding .text-field-input")
+  private WebElement sidebarInputField;
+
+  @FindBy(css = ".infobox-builder-sidebar-header-icon-back")
+  private WebElement backArrowButton;
+
+  @FindBy(css = ".infobox-builder-buttons")
+  private WebElement componentsButtonsWrapper;
+
+  @FindBy(css = "#go-to-source")
+  private WebElement goToSourceButton;
+
+  @FindBy(css = ".pi-header")
+  private WebElement infoboxHeader;
+
+  @FindBy(css = ".infobox-builder-button")
+  private List<WebElement> componentsButtons;
+
+  public Sidebar() { super(); }
+
+  public boolean isSidebarInputFieldFocused() {
+    return sidebarInputField.equals(driver.switchTo().activeElement());
+  }
+
+  public boolean isInputFieldPresent() {
+    wait.forElementVisible(sidebarInputField);
+
+    return sidebarInputField.isDisplayed();
+  }
+
+  public boolean isTitleUsingArticleName() {
+    wait.forElementClickable(sidebarCheckbox);
+
+    return sidebarCheckbox.isSelected();
+  }
+
+  public Sidebar setTitleToUseArticleName() {
+    wait.forElementClickable(sidebarCheckbox);
+
+    if (!sidebarCheckbox.isSelected()) {
+      sidebarCheckbox.click();
+    }
+
+    return this;
+  }
+
+  public boolean areAddButtonsPresent() {
+    wait.forElementVisible(componentsButtonsWrapper);
+
+    return componentsButtonsWrapper.isDisplayed();
```

Comment:
Same as before - return true will do the job, but I would advise you to build it in a way that method returns true or false (depending on the state), or if you want to give it time to be visible, you can build a wait (with 5s timeout) to return bool after max 5s